sympy/stats/frv.py

```diff
@@ -298,7 +298,7 @@ def compute_characteristic_function(self, expr):
 
         return Lambda(t, sum(exp(I*k*t)*v for k,v in d.items()))
 
-    def integrate(self, expr, rvs=None):
+    def integrate(self, expr, rvs=None, evaluate=True):
```

Comment:
Is `evaluate` flag is meant to be used here somehow? It's strange to have it accepted and not used.

Reply:
This is needed to match the signature on line 581 of `rv.py`, otherwise it would cause a keyword error.